ec2/roles/jboss-widlfly/.travis.yml

```diff
@@ -0,0 +1,29 @@
+---
+language: python
+python: "2.7"
```

Comment:
do we want 2.7 for some reason? We really should be only using 3+ by this point

Reply:
No. Its from the boilerplate code. Removing it right away. Good catch.

Reply:
^ meant for the bsd question. 
No. All these are boilerplate code. I will update them

Reply:
Updated the version to 3.11. That is what I am using but open to suggestion